alibi/datasets/default.py

```diff
@@ -51,7 +45,7 @@ def load_cats(target_size: tuple = (299, 299), return_X_y: bool = False) -> Unio
     (data, target)
         Tuple if ``return_X_y=True``.
     """
-    tar = tarfile.open(fileobj=BytesIO(pkgutil.get_data(__name__, "data/cats.tar.gz")),  # type: ignore[arg-type]
+    tar = tarfile.open(fileobj=BytesIO(pkgutil.get_data(__name__, "../data/cats.tar.gz")),  # type: ignore[arg-type]
```

Comment:
Just to check, `__name__` here would refer to the `datasets` package (in previous version it would be `alibi` package)? And the relative reference seems to resolve to the correct location of the data?

Reply:
yep